.github/workflows/prerelease.yml

```diff
@@ -88,11 +88,24 @@ jobs:
     needs: [validate, test-nix, snyk, test-integration-nix]
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
       - name: Login to DockerHub
         uses: docker/login-action@v1
         with:
           username: ${{ secrets.OHAI_DOCKER_HUB_ID }}
           password: ${{ secrets.OHAI_DOCKER_HUB_PASSWORD }}
+      - name: Create CHANGELOG.md
+        uses: heinrichreimer/github-changelog-generator-action@v2.2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issues: true
```

Comment:
i would not add issues. We will end up populating the changelog with things that are not related to code changes like [this one ](https://github.com/newrelic/nri-docker/issues/47)

Reply:
I can try to add issues but only those with labels (`bug`, `enhancement`, etc.) and see how it comes.